code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -562,10 +562,6 @@
 /obj/item/robot_module/drone/respawn_consumable(mob/living/silicon/robot/R)
 	var/obj/item/reagent_containers/spray/cleaner/C = locate() in modules
 	C.reagents.add_reagent("cleaner", 3)
-
-	var/obj/item/lightreplacer/LR = locate() in modules
-	LR.Charge(R)

```

Comment:
This is removed because it's handled on the recharge,r as disgusting as that is. It would be *better* if it was handled here, in `respawn_consumable`, but that'd require a larger refactor of borg rechargers and appropriately using `respawn_consumable`, which I really didn't feel like doing in this PR.